library/ecjpake.c

```diff
@@ -820,6 +820,8 @@ static const unsigned char ecjpake_test_password[] = {
     0x65, 0x73, 0x74
 };
 
+#if !defined(MBEDTLS_ECJPAKE_ALT)
+
```

Comment:
I believe this guard could be extended to also cover the function: `ecjpake_lgc`, as it does not appear to be used in any tests that would still run when `MBEDTLS_ECJPAKE_ALT` is defined.

Reply:
`ecjpake_lgc` is in use for the not-skipped tests that test the implementation against itself ("#1 random handshake")

Reply:
My apologies, It looks like I misread where the lower guard started.